pages/common/awk.md

```diff
@@ -1,36 +1,36 @@
 # awk
 
 > A versatile programming language for working on files.
-> More information: <https://github.com/onetrueawk/awk>.
+> More information: <https://manned.org/awk.1>.
```

Comment:
The project page is preferable to a man page. Let's revert this change and keep it as it was.

Reply:
But there is no language description directly available via this suggested url. It may confuse users.

Reply:
I'm not sure what you're referring to here. Could you clarify please @EmilySeville7cfg?